app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java

```diff
@@ -162,6 +168,18 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
 
         mId = eatery.id();
 
+        for (AllEateriesQuery.SwipeDatum swipeDatum : eatery.swipeData()) {
```

Comment:
Was talking to designers yesterday and I think the plan is to do 30min/hour long increments from 6am to 3am the next day. Is this what the backend supports/does it still return data if there were no swipes in that interval? If not we might want to do some padding of swipedata here to mSwipeDataList by adding intervals with density=0 at the beginning of the day and such

Reply:
Yep, I'm gona ask Conner tmrw about what backend returns for future times that haven't happened yet

Reply:
talked to backend, so for time intervals that have not happened yet they use historical data, so swipe density ~ wait time high wouldn't be 0. I also confirmed right now(4:30PM) that all future times have some value.